### PR TITLE
modified shortbutton, removed extra border from profilepicture

### PR DESCRIPTION
**Pull request by Pullisepop**
-added a border to ButtonShort according to Figma design
-removed double border around ProfilePicture2
-cleaned up code on editProfilePage and ProfilePage

### CODE REVIEW

File: src/pages/EditProfilePage/EditProfilePage.css
Comment (tommasoghisini) [inline src/pages/EditProfilePage/EditProfilePage.css:1]: only thing to fix is edit name which looks like this:

Screenshot 2023-12-10 at 11 54 00 AM

**Review by tommasoghisini — CHANGES_REQUESTED** [review]
see comment

good job!

File: src/pages/EditProfilePage/EditProfilePage.css
Comment (Pullisepop) [reply src/pages/EditProfilePage/EditProfilePage.css:1]: Ok, I see. The buttons move when edit name is pressed. Will fix.

Comment (Pullisepop) [reply src/pages/EditProfilePage/EditProfilePage.css:1]: I have pushed the changes now.

**Pull request merged**